03-explanation/02-best-practices/04-agentic-mode.mdx

```diff
@@ -1,26 +1,9 @@
 ---
-title: "Agentic Mode"
+title: "Effective Agentic Instructions"
```

Comment:
Can we then rename the file to `04-effective-agentic-instructions`

Reply:
done